.travis.yml

```diff
@@ -10,6 +10,8 @@ stages:
   - name: integration_test
   - name: package
   - name: publish
+install:
+  - cp integration_tests/test_config.ini ~/.jccli.ini
```

Comment:
you are testing with the API key env var now, correct?  do you still need this?

Reply:
Correct. Just deleted